aten/src/ATen/native/sparse/SparseTensorMath.cpp

```diff
@@ -246,7 +233,7 @@ SparseTensor& true_divide_out_sparse_zerodim(
     Tensor dividend_tmp = dividend;
     result.resize_as_(dividend_tmp);
     auto indices = result._indices();
-    indices.resize_as_(dividend_tmp.indices());
+    indices.resize_as_(dividend_tmp._indices());
```

Comment:
is this an efficiency change?  (so you don't have to create a view?).

Reply:
Yes. This was actually a typo in the true_divide PR. The sparse div implementation consistently uses _indices(). The difference between indices() and _indices() is documented here: https://github.com/pytorch/pytorch/blob/43c9cc7a9cb19ac40c4296a7a7c0e1788bf5d288/aten/src/ATen/native/native_functions.yaml#L3283.